Lesson34/feyza/lesson34-app/src/App.js

```diff
@@ -0,0 +1,40 @@
+import { List } from "./components/List";
+import { Navbar } from "./components/Navbar";
+import { PerformanceState } from "./components/PerformanceState";
+import { useState, useContext } from "react";
+import { todoData } from "./data";
+import { UserProvider } from "./userContext";
+import { Account } from "./components/Account";
+import { UserContext } from "./components/UserContext";
+import { LogIn } from "./components/Login";
+import "./App.css";
+
+export const App = () => {
+  const [todos, setTodos] = useState(todoData);
+  const user = useContext(UserContext);
+
+  const loggedInUser = {
+    isLoggedInUser: true,
+    username: "johndoe01",
+  };
+  const loggedOutUser = {
```

Comment:
Move this variable to index.js file (main file in the src folder). You'll also need to move UserProvide there like this:
```
const root = ReactDOM.createRoot(document.getElementById("root"));
root.render(
  <React.StrictMode>
    <UserProvider initialState={loggedOutUser}>
      <App />
    </UserProvider>
  </React.StrictMode>
);
```

It's important to move the UserProvider to the index.js file since otherwise you cannot access data from the context in the App component.